### DISCUSSION TITLE
Test skipped as passed

### DISCUSSION THREAD

**Discussion by gambatti** [MicroFocus/bdd2octane#3] [Q&A]
Hey guys, I’m trying to use the component but something might be wrong.
Some of my tests were skipped and they should be faild in the report, but all of them are as passed. 
The reason of the fail was because the tests were not implemented yet. 
Is that a problem or I am doing something wrong?
Thank you.

**Comment by ghzhou** [comment]
Could you please paste the sample skipped test here?

**Comment by Alex-K-COBA** [comment]
I have the same problem that my pretty junit xml report contains failed steps but the bdd2octane.jar creates a XML file for Octane that only contains passed steps and scenarios.

**Reply by nissimshitrit** [reply]
Can you please share you reports xml and feature ?

**Reply by Alex-K-COBA** [reply]
Hi, thank you for the fast response. I added now the new bdd2octane jar version and it works now.
